website/docs/d/vm_sizing_policy.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vm_sizing_policy"
+sidebar_current: "docs-vcd-datasource-vm-sizing-policy"
+description: |-
+  Provides a vCloud Director VM sizing policy data source. This can be
+  used to read VM sizing policy.
+---
+
+# vcd\_vm\_sizing\_policy
+
+Provides a vCloud Director VM sizing policy data source. This can be
+used to read VM sizing policy.
+
+Supported in provider *v3.0+*
```

Comment:
Should we also state from which VCD version it's supported?

Reply:
added